common/lib/src/realm_types.dart

```diff
@@ -316,17 +313,14 @@ enum SyncConnectionErrorCode {
   switchToFlxSync(113),
 
   /// Connected with wrong wire protocol - should switch to PBS
-  switchToPbs(114);
+  switchToPbs(114),
+
+  unknown(9999);
```

Comment:
no api doc

Reply:
and all others are missing api docs

Reply:
What should it be. "Unknown error code" ?